plugins/ruby-foreman-datacenter/debian/rules

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+# Sample debian/rules that uses debhelper.
+# This file was originally written by Joey Hess and Craig Small.
+# As a special exception, when this file is copied by dh-make into a
+# dh-make output file, you may use that output file without restriction.
+# This special exception was added by Craig Small in version 0.37 of dh-make.
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+PLUGIN = foreman_datacenter
+
+build:
+	cp cache/* /usr/share/foreman/vendor/cache/
```

Comment:
This fails because the cache doesn't exist.

```
17:24:46 cp cache/* /usr/share/foreman/vendor/cache/
17:24:46 cp: cannot create regular file '/usr/share/foreman/vendor/cache/': No such file or directory
17:24:46 debian/rules:15: recipe for target 'build' failed
17:24:46 make: *** [build] Error 1
17:24:46 dpkg-buildpackage: error: debian/rules build gave error exit status 2
```